packages/rest/src/lib/RequestManager.ts

```diff
@@ -41,6 +42,10 @@ export interface RequestData {
 	 * The body to send to this request
 	 */
 	body?: unknown;
+	/**
+	 * Whether to append JSON data to form data isntead of `payload_json` when sending attachments
+	 */
+	dontUsePayloadJSON?: boolean;
```

Comment:
Do we want to keep this name? The `dont` is killing me 😂 @discordjs/the-big-4

Reply:
I wanted to comment on the name too lol. Probably not.

Reply:
Oh good, I was on the fence about changing this to be a more opt in feel instead of opting out (especially considering it's literally opt in)